### PR TITLE
1556-gndCombination2

### PR DESCRIPTION
**Pull request by TobiasNx**

### CODE REVIEW

File: src/test/resources/alma-fix/990015940770206447.json [hunk 1]
```diff
@@ -202,7 +202,7 @@
       "id" : "https://d-nb.info/gnd/2106160-9",
       "label" : "Fachinformationszentrum Karlsruhe",
       "type" : [ "CorporateBody" ],
-      "altLabel" : [ "Fachinformationszentrum", "Gesellschaft für Wissenschaftlich-Technische Information mbH", "Fachinformationszentrum Karlsruhe, Gesellschaft für Wissenschaftlich-Technische Information mbH", "FIZ KA", "FIZ-KA", "FIZ Karlsruhe", "Fachinformationszentrum", "Gesellschaft für Wissenschaftlich-Technische Information mbH", "Fachinformationszentrum Karlsruhe, Gesellschaft für Wissenschaftlich-Technische Information mbH", "FIZ KA", "FIZ-KA", "FIZ Karlsruhe" ]
+      "altLabel" : [ "Fachinformationszentrum (Karlsruhe)", "Gesellschaft für Wissenschaftlich-Technische Information mbH", "Fachinformationszentrum Karlsruhe, Gesellschaft für Wissenschaftlich-Technische Information mbH", "FIZ KA", "FIZ-KA", "FIZ Karlsruhe (1988-2009)", "Fachinformationszentrum (Karlsruhe)", "Gesellschaft für Wissenschaftlich-Technische Information mbH", "Fachinformationszentrum Karlsruhe, Gesellschaft für Wissenschaftlich-Technische Information mbH", "FIZ KA", "FIZ-KA", "FIZ Karlsruhe (1988-2009)" ]
```
Comment (dr0i) [inline src/test/resources/alma-fix/990015940770206447.json:205]: There are redundant values. Can you make them unique?

**Comment by TobiasNx** [comment]
> There are redundant values. Can you make them unique? 

Done.

**Comment by TobiasNx** [comment]
Also introduced the GND pattern for labels if 600. Found no tests for that. But it does not break the tests pattern.

**Review by dr0i — APPROVED** [review]
thx +1

**Pull request merged**